utils/logger/logger.go

```diff
@@ -1,222 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See License.txt for license information.
-
-// this is a new logger interface for mattermost
```

Comment:
I don't think anyone's really motivated to work on this anymore. The task is in my backlog, but only because I want to do some minor refactors to make things race-free. If on the off-chance anyone does decide they want to re-use this code, they can get it from the history.